src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -24,6 +25,8 @@ import (
 
 const (
 	PortmapFailure = 50
+
+	CF_IPCOUNTRY = "Cf-Ipcountry"
```

Comment:
Are we following "general convention" now and using MixedCaps instead of uppercase + underscore for naming constants?